scripts/setup-android-sdk.sh

```diff
@@ -10,9 +10,9 @@ set -e -u
 
 ANDROID_SDK_FILE=commandlinetools-linux-${TERMUX_SDK_REVISION}_latest.zip
 ANDROID_SDK_SHA256=0bebf59339eaa534f4217f8aa0972d14dc49e7207be225511073c661ae01da0a
-if [ "$TERMUX_NDK_VERSION" = "26b" ]; then
+if [ "$TERMUX_NDK_VERSION" = "27" ]; then
 	ANDROID_NDK_FILE=android-ndk-r${TERMUX_NDK_VERSION}-linux.zip
-	ANDROID_NDK_SHA256=ad73c0370f0b0a87d1671ed2fd5a9ac9acfd1eb5c43a7fbfbd330f85d19dd632
+	ANDROID_NDK_SHA256=2f17eb8bcbfdc40201c0b36e9a70826fcd2524ab7a2a235e2c71186c302da1dc
 elif [ "$TERMUX_NDK_VERSION" = 23c ]; then
```

Comment:
Any reason this pull keeps 23c around, rather than switching to keeping 26b as the backup and 27 as the new default?

Reply:
It is the last NDK to support `-march=armv7-a -mfpu=vfpv3-d16`

Reply:
Some packages are still using it for that?

Reply:
All existing packages are built with LDFLAGS `-mfpu=neon` so no. This is only kept to extend hardware support for Termux. I am pretty much the only maintainer that still uses it.

Reply:
OK, what do you use 23c for, some old armv7 devices?

@licy183, would it make sense to keep 26b around also as a backup or no need?

Reply:
Emmm... I think there is no need to keep NDK 26b.

Reply:
Wasn't 23c useful for compiling some packages in termux-user-repository as well?

Reply:
Hmm, github is removing everything older than NDK 26 from their CI images, actions/runner-images#10342.

Reply:
Time to drop out support of `armv7-a`, `aarch64` aka `armv8` seem common now. We need support `riscv64` and `riscv128` in long future.